WeakAurasOptions/BuffTrigger2.lua

```diff
@@ -578,6 +578,62 @@ local function GetBuffTriggerOptions(data, triggernum)
         WeakAuras.Add(data)
       end
     },
+    use_isBossDebuff = {
+      type = "toggle",
+      name = function(input)
+        local value = trigger.use_isBossDebuff
+        if value == nil then return L["Is Boss Debuff"]
+        elseif value == false then return "|cFFFF0000 "..L["Negator"].." "..L["Is Boss Debuff"]
+        else return "|cFF00FF00"..L["Is Boss Debuff"] end
+      end,
+      width = WeakAuras.doubleWidth,
+      order = 64.1,
+      hidden = function() return not (trigger.type == "aura2" and trigger.unit ~= "multi" and not IsSingleMissing(trigger)) end,
+      get = function()
+        local value = trigger.use_isBossDebuff
+        if value == nil then return false
+        elseif value == false then return "false"
+        else return "true" end
+      end,
+      set = function(info, v)
+        if v then
+          trigger.use_isBossDebuff = true
+        else
+          local value = trigger.use_isBossDebuff
+          if value == false then trigger.use_isBossDebuff = nil
+          else trigger.use_isBossDebuff = false end
+        end
+        WeakAuras.Add(data)
+      end
+    },
+    use_castByPlayer = {
+      type = "toggle",
+      name = function(input)
+        local value = trigger.use_castByPlayer
+        if value == nil then return L["Cast by Player Character"]
+        elseif value == false then return "|cFFFF0000 "..L["Negator"].." "..L["Cast by Player Character"]
+        else return "|cFF00FF00"..L["Cast by Player Character"] end
+      end,
+      width = WeakAuras.doubleWidth,
```

Comment:
```Lua
desc = L["Only Match auras cast by a player (not an npc)"],
```

Reply:
Yea, description will definitely be helpful here. I would like to see its implementation as in the case of "Own only" (two descriptions; a couple of lines below ↓)